Assets/Prefab/UI/HUD Canvas.prefab

```diff
@@ -1,5 +1,139 @@
 %YAML 1.1
 %TAG !u! tag:unity3d.com,2011:
+--- !u!1 &164296227500800776
```

Comment:
dont display wave timer on the UI for anything other than debug purposes...

Reply:
ok, this was a design choice that was totally alien to me
if we want that kind of design this si fine as a warning that enemies are about to spawn